onnxruntime/core/session/inference_session.h

```diff
@@ -305,12 +305,20 @@ class InferenceSession {
                                    std::vector<OrtValue>* p_fetches,
                                    const std::vector<OrtDevice>* p_fetches_device_info = nullptr);
 
+  /*
```

Comment:
Can you follow the existing documentation format? Params need to be documented too. Here's an [example](https://github.com/microsoft/onnxruntime/blob/e1ca8ee6d4a188f01efd4083d6f4273a3ef901aa/onnxruntime/core/session/inference_session.h#L322-L331). You may refer to another Run func to avoid repetition as seen [here](https://github.com/microsoft/onnxruntime/blob/e1ca8ee6d4a188f01efd4083d6f4273a3ef901aa/onnxruntime/core/session/inference_session.h#L335-L339).